### PR TITLE
README: Add a short section about History

### PR DESCRIPTION
**Pull request by certik**

### CODE REVIEW

**Review by awvwgk — APPROVED** [review]
Thanks, looks good to me.

File: README.md [hunk 1]
```diff
@@ -30,6 +30,35 @@ Further updates are planned...
 
 The Minpack source code and related files and documentation are distributed under a permissive free software [license](https://github.com/fortran-lang/minpack/blob/HEAD/LICENSE.txt) (BSD-style).
 
+### History
+
+Minpack has been developed in 1980 by Jorge J. Moré, Burton S. Garbow, Kenneth
+E.  Hillstrom and other contributors as listed on page 8 of the [User Guide for
+MINPACK-1](http://cds.cern.ch/record/126569/files/CM-P00068642.pdf).
+
+Since 2012 Ondřej Čertík has maintained a github repository for minpack with
```
Comment (milancurcic) [inline README.md:39]:
```suggestion
Since 2012 Ondřej Čertík has maintained a GitHub repository for minpack with
```

File: README.md [hunk 2]
```diff
@@ -30,6 +30,35 @@ Further updates are planned...
...
+many contributions from Carlos Une and Zuo Zhihua.
+
+In 2021 Jacob Williams started a new minpack repository at github and
```
Comment (milancurcic) [inline README.md:42]:
```suggestion
In 2021 Jacob Williams started a new minpack repository at GitHub and
```

File: README.md [hunk 3]
```diff
@@ -30,6 +30,35 @@ Further updates are planned...
...
+translated all files from fixed form to free form and other modernizations.
+
+We have discussed at https://github.com/fortran-lang/minpack/issues/8 which
+version to use as the community maintained fortran-lang version and decided to
+use the latter repository, which now became the fortran-lang version. We have
+or are in the process of porting improvements from the former repository.
```
Comment (milancurcic) [inline README.md:48]:
```suggestion
use the latter repository, which became the fortran-lang version.
We are in the process of porting improvements from the former repository.
```
```suggestion
use the latter repository, which now became the fortran-lang version. We have
or are in the process of porting improvements from the former repository.
```

**Review by milancurcic — APPROVED** [review]
Looks good, thanks, I added very minor editorial suggestions.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (certik) [reply README.md:48]: @milancurcic do you prefer this line, or the one you proposed below?

[the remark this answers is not in the document]
Comment (milancurcic) [reply README.md:48]: I like this one better as it's more time invariant. Will close mine, but would still like the "now" removed from the sentence before this one.

File: README.md [hunk 4]
```diff
@@ -30,6 +30,35 @@ Further updates are planned...
...
+
+Since 2012 Ondřej Čertík has maintained a github repository for minpack with
+many contributions from Carlos Une and Zuo Zhihua.
+
+In 2021 Jacob Williams started a new minpack repository at github and
+translated all files from fixed form to free form and other modernizations.
+
+We have discussed at https://github.com/fortran-lang/minpack/issues/8 which
+version to use as the community maintained fortran-lang version and decided to
+use the latter repository, which now became the fortran-lang version. We have
```
Comment (milancurcic) [inline README.md:47]:
```suggestion
use the latter repository, which became the fortran-lang version. We have
```

**Comment by certik** [comment]
Thanks @awvwgk and @milancurcic. Milan, let me know if it looks good now to merge.

**Comment by milancurcic** [comment]
Looks great! Will merge, thank you.

**Pull request merged**